src/components/BrowserEventList/EventDescription/Selector.tsx

```diff
@@ -28,23 +30,25 @@ export function Selector({ selector, onHighlight }: SelectorProps) {
   }
 
   return (
-    <strong
+    <Badge
```

Comment:
Fun fact: you need to create a custom component for this since `@radix-ui/themes` doesn't work in the browser extension. 😞

<img width="687" height="807" alt="Image" src="https://github.com/user-attachments/assets/42543f91-09cb-42d0-a9d5-8fd2a1a13c1d" />